bin/civi-download-tools

```diff
@@ -743,6 +743,10 @@ pushd $PRJDIR >> /dev/null
   make_link "$PRJDIR/bin" "../extern/phpunit8/phpunit8.phar" "phpunit8"
   make_link "$PRJDIR/bin" "../extern/phpunit9/phpunit9.phar" "phpunit9"
 
+  [ ! -d "$HOME/.drush" ] && mkdir "$HOME/.drush"
+  [ ! -d "$HOME/.drush/command" ] && mkdir "$HOME/.drush/command"
+  make_link "$HOME/.drush/command" "$PRJDIR/extern/drush-lib/backdrop" "backdrop"
```

Comment:
Currently, `bin/drush8` is [a wrapper that applies `--include="$PRJDIR/extern/drush-lib"`](https://github.com/civicrm/civicrm-buildkit/blob/master/src/drush/drush8.tmpl). This allows the `buildkit` directory to be self-contained (ie you can clone/run/delete without any effect on system state; you can have multiple copies without affecting each other).

Did something change in `drush` or `drush-backdrop` to make `HOME` mandatory?

Reply:
@totten no i just seemed to manage to clobber the wrapper completely in my testing of drush test phars it seems